docs/docs/api/@dxos/client/classes/HaloProxy.md

```diff
@@ -17,16 +17,16 @@ Arguments:
 ## Properties
 ### [invitationsUpdate](https://github.com/dxos/dxos/blob/main/packages/sdk/client/src/packlets/proxies/halo-proxy.ts#L55)
 Type: <code>Event&lt;void | [CancellableInvitationObservable](/api/@dxos/client/interfaces/CancellableInvitationObservable)&gt;</code>
-### [profileChanged](https://github.com/dxos/dxos/blob/main/packages/sdk/client/src/packlets/proxies/halo-proxy.ts#L56)
+### [identityChanged](https://github.com/dxos/dxos/blob/main/packages/sdk/client/src/packlets/proxies/halo-proxy.ts#L56)
```

Comment:
fyi (I will document this) these docs in `/docs/docs/api` are auto-generated from `pnpm nx apidoc docs`

Reply:
Should I run it myself, or do you want to go through the docs before?